README.md

```diff
@@ -9,6 +9,11 @@ Just add `dashed` to your Gemfile and then run `bundle install`.
 Dashed has to be run with root permissions in order to listen to your network
 devices to pick up your dash button.
 
+## Preparing the buttons
+
+In order to use your [dash buttons](http://www.amazon.com/b/?node=10667898011&sort=date-desc-rank&lo=digital-text) with this library, you will need to set them up using Amazon's setup process. While you can automatically order products using the buttons, if you'd like to prevent the automatic ordering of a product, simply stop the  registration process before choosing a product.
```

Comment:
Thoughts on pointing people to this instead:
`http://www.amazon.com/oc/dash-button` ?